app/src/main/java/com/droidhats/campuscompass/views/MapFragment.kt

```diff
@@ -288,8 +295,8 @@ class MapFragment : Fragment(), OnMapReadyCallback, GoogleMap.OnMarkerClickListe
 
             //Calculating the center of the polygon to use for it's location.
             // This won't be necessary once we hold the Buildings in a common class
-            var centerLat: Double = 0.0
-            var centerLong: Double = 0.0
```

Comment:
Please do not remove types to our variables, it'll make it harder to debug if this becomes a habit

Reply:
Sorry!